### ISSUE TITLE
Avoid spewing warnings from dependent libraries

### ISSUE THREAD

**Issue by jimbobhickville** [ToolsForHumans/zag#42]
Alembic has renamed some columns and spits out a warning every time the Zag migrations are run about a field being deprecated. We should update our supported alembic version and change the parameter names to be forwards-compatible.

```
UserWarning: Argument 'name' is now named 'constraint_name' for method create_foreign_key()
```

**Comment by jimbobhickville** [comment]
sqlalchemy 1.3 also deprecated the `convert_unicode` parameter:

```
SADeprecationWarning: The create_engine.convert_unicode parameter and corresponding dialect-level parameters are deprecated, and will be removed in a future release.  Modern DBAPIs support Python Unicode natively and this parameter is unnecessary.
```

**Comment by jimbobhickville** [comment]
Fixed by #46
